CHANGES.txt

```diff
@@ -27,6 +27,9 @@ Changes
 Fixes
 =====
 
+- Fixed a regression which caused ``IS NOT NULL`` predicates on columns of type
```

Comment:
isn't this also about `IS NULL` ?

Reply:
``IS NULL`` is optimized to use the `_field_names` which are part of the index. So it wasn't affected.

We could also optimize this for ``IS NOT NULL`` later on.